core/src/main/resources/xml/xsl/adapter2mermaid_working_version.xsl

```diff
@@ -0,0 +1,987 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
welke van deze twee mermaid xslt's is de juiste? je kan beter de andere weglaten dat is alleen maar verwarrend